cosmos/sql/tools/java/ycsb/recipes/read/try-it-read/azuredeploy.json

```diff
@@ -283,6 +318,21 @@
                     },
                     "fieldcount": {
                         "value": "[parameters('fieldcount')]"
+                    },
+                    "appInsightConnectionString": {
+                        "value": "[parameters('appInsightConnectionString')]"
+                    },
+                    "pointOperationLatencyThresholdInMS": {
+                        "value": "[parameters('pointOperationLatencyThresholdInMS')]"
```

Comment:
These parameters are for internal use only. As discussed, let's not expose them in the recipes.

Reply:
ok sure. removed.